### PR TITLE
Add PowerDNS-Admin to Easypanel 🎉

### PR DESCRIPTION
**Pull request by migu2k**

### CODE REVIEW

File: templates/grafana/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      title: Project Name
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: grafana
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: grafana/grafana-oss:8.2.6
+    serviceTimezone:
```
Comment (deiucanta) [inline templates/grafana/meta.yaml:35]: You are requesting this data in the form, but you don't seem to use it in the template code.

File: templates/grafana/meta.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        default: "grafana/grafana-oss:8.2.6",
+      },
+      serviceTimezone: {
+        type: "string",
+        title: "Timezone",
+        default: "Europe/London",
+      },
+    },
+  },
+  logo: null,
```
Comment (deiucanta) [inline templates/grafana/meta.ts:37]: The logo was not detected since it's a JPG file. If should be PNG or SVG.

**Comment by deiucanta** [comment]
@migu2k Thanks for your PR!

Can you add the proper mounts to Grafana?

**Comment by migu2k** [comment]
> @migu2k Thanks for your PR!
> 
> Can you add the proper mounts to Grafana?

I've removed Grafana from the PR entirely for now. I'll rework it and create a separate PR at a later point in time.

**Comment by deiucanta** [comment]
@migu2k oh, didn't notice that :)

Can you then fix the conflicts? I can fix them, but the contribution will show as mine.

**Comment by migu2k** [comment]
Scrap that commit, I've messed up big time there fsr. I'd be thankful if you could fix the conflicts for me as I'm running a bit behind schedule atm. If not no worries, I'll fix them later this evening.

**Comment by deiucanta** [comment]
I merged it locally and fixed the conflicts. Thanks again!

**Pull request closed without merging**